reference/ambassador-with-gke.md

```diff
@@ -115,9 +133,24 @@ spec:
 ```
 
 Modify the ingress from before:
+
 ```yaml
 apiVersion: extensions/v1beta1
 kind: Ingress
+metadata:
+  name: basic-ingress
+  annotations:
+    networking.gke.io/managed-certificates: www-example-com
+spec:
+  backend:
+    serviceName: ambassador
+    servicePort: 8080
+```
+ or 
+
+```yaml
+apiVersion: extensions/v1
```

Comment:
This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.